osu.Game/Screens/Play/HUD/LegacyComboSplash.cs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public class LegacyComboSplash : Container<Container<Sprite>>, ISkinnableDrawable
+    {
+        [SettingSource("Side, where bursts will appear")]
+        public Bindable<Side> BurstsSide { get; } = new Bindable<Side>(Side.Random);
```

Comment:
Not sure about making this the new sort of setting rather than hooking it up to the old setting flow that reads from `skin.ini`. This is kind of uncharted territory still.

Reply:
Afaik there was no way to change this in stable (for osu! and osu!taiko) and there is no setting for this in `skin.ini`. I want to implement this here to make the component more flexible, so decided to use "modern" setting.

A mania-specific option for this exists, but it doesn't suit the behaviour implemented here (anchors are screen's sides, not playfield's ones):
![image](https://user-images.githubusercontent.com/53872073/162337430-9eda314a-ac7b-45c4-819a-19f8e1dd6eee.png)
Should i use it instead of "modern" setting anyway?